blockchain_api/app.js

```diff
@@ -58,11 +62,16 @@ app.use(bodyParser.urlencoded({
 }));
 // set secret variable
 app.set('secret', 'thisismysecret');
-app.use(function(req, res, next) {
-	logger.debug(' ------>>>>>> new request for %s',req.originalUrl);
-	req.username = 'user1';
-	req.orgname = 'Org1';
-	return next();
+app.use(async function(req, res, next) {
+	try {
+		var user = await enrollApiUser.enrollApiUser();
```

Comment:
This function is going to run before each request to api
Isn't it better to move it  to start_chaincodes.js file?

Reply:
Inside it checks if the user is enrolled already and just exits in this case. 
If not - it actually enrolls the user.